.github/workflows/purge-test-resources.yaml

```diff
@@ -21,6 +21,8 @@ permissions:
   contents: read # Required for actions/checkout
 
 on:
+  # Enable manual trigger
+  workflow_dispatch:
```

Comment:
FYI is this is going to create an issue on failure when run manually. If you want to avoid that, you could change the condition [here](https://github.com/radius-project/radius/pull/7622/files#diff-9270fafc31ef4b50d35d1295e9f99f96013853c295c29c92c387492f62cd570dL102) to skip the issue for a manual run.

Reply:
Good catch, will update to skip on a manual run